examples/interpreter/physl.cpp

```diff
@@ -160,6 +170,8 @@ int handle_command_line(int argc, char* argv[], po::variables_map& vm)
                 "of the created execution tree as a Newick tree to a file")
             ("transform,t", po::value<std::string>(),
                 "file to read transformation rules from")
+            ("no-ast-env,e", po::value<std::string>()->implicit_value("<none>"),
```

Comment:
The default typically is to read from STDIN and read as is. Maybe I suggest providing a flag that does the opposite and reads base64 if the flag is present?

Reply:
@parsa great question! the environment variable is meant to hold the IR and enable a cloud deployment of phylanx to a remote system for batch execution. a user can tell the python-side of phylanx to deploy the phylanx interpreter, as a container, using a cloud scheduler. to make that work without having to manage socket handshakes between the python-side and the remote system, the IR should be deployed with the container. packing IR into an environment variable is arguably tacky, but makes it possible to ship IR with the container job description for remote deployments.

Reply:
@hkaiser @parsa i've been thinking if the environment variable is too tacky, a command line flag that accepts base64 encoded IR would also work.

Reply:
@parsa I've added in the command line argument would "b64-physl" work?

Reply:
@ct-clmsn Sure. Or `--base64,b` or something users can remember.

Reply:
@parsa thanks! will add that into the fix.